packages/lisk-core/src/channels/event_emitter.js

```diff
@@ -0,0 +1,62 @@
+const Event = require('../event');
+const Action = require('../action');
+const BaseChannel = require('./base');
+
+module.exports = class EventEmitterChannel extends BaseChannel {
+	constructor(moduleAlias, events, actions, bus, options = {}) {
+		super(moduleAlias, events, actions, options);
+		this.bus = bus;
+		this.actionMap = {};
+	}
+
+	async registerToBus() {
```

Comment:
ChildProcessChannel class uses its rpcClient to make a call to 'registerChannel method. Why not following the same approach?